website/docs/kubernetes/openshift/creating-an-openshift-local-cluster.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 1
+title: OpenShift Local
+description: Creating an OpenShift Local instance
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes, openshift]
+tags: [migrating-to-kubernetes, openshift]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating an OpenShift Local instance
+
+[Red Hat OpenShift Local](https://developers.redhat.com/products/openshift-local/overview) manages a minimal OpenShift or MicroShift cluster on your workstation for local development and testing.
+
+With Podman Desktop and the OpenShift Local extension, you can manage your OpenShift Local instances.
+
+#### Prerequisites
+
+- [Register a Red Hat account](https://developers.redhat.com/register).
+
+#### Procedure
+
+1. Install the _OpenShift Local_ extension: on to **Dashboard**, click **OpenShift Local <icon icon="fa-solid fa-download" size="lg" />**.
+1. Install the OpenShift Local binaries, when on the **Dashboard**, you see _Podman Desktop was not able to find an installation of OpenShift Local_.
+
+   <Tabs groupId="operating-systems">
+   <TabItem value="win" label="Windows">
+
+   1. In the **OpenShift Local** tile, click **<icon icon="fa-solid fa-rocket" size="lg" /> Install**.
+   1. When prerequisites are missing, follow the instructions.
+   1. In the **Red Hat OpenShift Local** screen, click **Yes** to start the installation.
+   1. Follow the installation program instructions.
+   1. Reboot to finalize system changes.
+
+   </TabItem>
+   <TabItem value="mac" label="macOS">
+
+   1. In the **OpenShift Local** tile, click **<icon icon="fa-solid fa-rocket" size="lg" /> Install**.
+   1. When prerequisites are missing, follow the instructions.
+   1. In the **Red Hat OpenShift Local** screen, click **Yes** to start the installation.
+   1. Follow the installation program instructions.
+   1. Reboot to finalize system changes.
+
+   </TabItem>
+   <TabItem value="linux" label="Linux">
+
+   1. Go to the [Red Hat OpenShift local download page](https://console.redhat.com/openshift/create/local).
+   1. Select your platform.
+   1. Click **Download OpenShift Local**.
+   1. Extract the archive.
+   1. Copy the `crc` binary to a directory in your`$PATH`, such as `/usr/local/bin`.
+   1. To configure your system, run the command:
+
+      ```shell-session
+      $ crc setup
+      ```
+
+   1. Exit and restart Podman Desktop.
+
+   </TabItem>
+   </Tabs>
+
+1. (Optionally) Review the extension settings in **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Preferences > Extension: Red Hat OpenShift Local**.
+1. On the **Dashboard**, click **Initialize and start**.
```

Comment:
Initialize and start never asked me (Windows) to select a preset ? Is this new ?

Reply:
It does only ask if it is not already set in your settings. It asked to me on Windows, once :).

Reply:
It only happens when you do a new install (uninstall and try again).

Reply:
Right, this is an onboarding step. Not a recurring event.